lib/make/makefile.bacmet

```diff
@@ -0,0 +1,209 @@
+ACCEPTED_SUFFIXES =
+
+# DIAMOND_ALIGN_OPTS sets non-essential parameters for diamond alignment
+#
+# See the Diamond documentation available at GitHub:
+# https://github.com/bbuchfink/diamond. 
+#
+# Look particularly at options for sensitivity, scoring and memory and
+# performance.
+#
+# To set options, override this by inserting a row similar to the one below but
+# with whatever options you'd like to add in your Makefile *after* you include
+# this file.
+DIAMOND_ALIGN_OPTS =  
+
+# Output format of the DIAMOND results. The default value refers to a tabular file.
+# For more format options see https://github.com/bbuchfink/diamond.
+OUTFMT = 6
+
+# Download HTTP URL for the BacMet protein database
+# In case you are not using ftp, you can override this macro and paste here the https URL.
+HTTP = http://bacmet.biomedicine.gu.se/download/BacMet2_EXP_database.fasta
+# IMPORTANT: databases are updated recurrently, so always check the availability of the http
+
+# Download HTTPS URL for the bacmet.py script, necessary for the downstream analyses
+HTTPS = https://gist.github.com/08ab264f0ea127cc242d99bc3844c743.git 
+# IMPORTANT: bacmet.py needs to be placed inside the directory that has the DIAMOND output
+# so run mirror_bacmet.py into the directory that contains .bacmet.sort.best.m8 files
+# N.B.: the python script "bacmetg.py" requires python 3.6*-3.8*
+
+# *** Internal ***
+
+# MAKECALL_DIAMOND_BACMET is a macro that defines what will be output to the .makecall
+# file, the file that records versions, file stamps, parameters etc.
+#
+# *Don't redefine!*
+MAKECALL_VERSION        = echo "`date +"%Y-%m-%d %H:%M:%S"`: $@ was made with `diamond --version`" > $@.makecall
+MAKECALL_PARAMS         = echo "	Called with parameters: $(DIAMOND_ALIGN_OPTS)" >> $@.makecall
+MAKECALL_INFILES        = echo "	Input files: $^ (`ls -lL $^|tr '\n' ','`)" >> $@.makecall
+MAKECALL_DB_BACMET      = echo "	Database: $(DIAMOND_DB_PATH)/$(DIAMOND_DB) (`ls -lL $(DIAMOND_DB_PATH)/$(DIAMOND_DB).dmnd`)" >> $@.makecall
+MAKECALL_DIAMOND_BACMET = $(MAKECALL_VERSION); $(MAKECALL_PARAMS); $(MAKECALL_DB_BACMET); $(MAKECALL_INFILES)
+MAKECALL_MAKEDB         = $(MAKECALL_VERSION); $(MAKECALL_INFILES)
+
+# *** misc targets ***
+
+# compress all .fastq files in the work directory.
+compress_all_fastqs: $(subst .fastq,.fastq.gz, $(wildcard *.fastq))
+
+# Interleave two fastq files (named %.R1.fastq and %.R2.fastq respectively).
+#
+# The target uses a python script -- interleave_fastq.py -- cloned from this
+# gist:
+#
+#   https://gist.github.com/rpucheq/cf50eeea794529ca6b0e
+#  *** Warning! ***
+# The aforementioned script works for bot .gzip and uncompressed files
+# but it has been written in PYTHON 2!!!
+#
+# Dependencies will automatically be unzipped if not present in unzipped format
+# thanks to the pattern above. Unzipped files will be deleted after
+# interleaving.
+interleave_all_fastq.gzs: $(subst .R1.fastq.gz,.intlv.fastq.gz,$(wildcard *.R1.fastq.gz))
```

Comment:
I'd prefer if you used `.r1.` and `.r2.` since that's the pattern used in all other makefiles in this repo. One has to create symlinks, but there's no way to handle all different ways forward and reverse read files are handled. Probably present in more places.